src/components/inputs/filled-icon-input/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { TextField, InputAdornment } from "@mui/material";
+
+const FilledIconInput = ({
+  icon = null,
+  placeHolder = "place holder",
+  isIconStart = true,
+  text = null,
+  handler,
+}) => {
+  // {type} prop can be: "filled", "outlined", or "standard"
```

Comment:
I can't see type prop in the component props.
would you please remove this comment if it's not needed?

Reply:
Yes, I forgot to delete that, let me do it.